File: typescript/26.ts
function removeDuplicates(nums: number[]): number {
  let n = 1
  for (let i = 0; i < nums.length - 1; i++) {
    if (nums[i] != nums[i + 1]) {
      nums[n] = nums[i + 1]
      n += 1
    }
  }
  return n
}

File: cpp/26.cpp
#include <vector>
using namespace std;

class Solution {
 public:
  int removeDuplicates(vector<int>& nums) {
    auto n = 1;
    for (auto i = 1; i < nums.size(); i++) {
      if (nums[i] != nums[i - 1]) {
        nums[n] = nums[i];
        n++;
      }
    }
    return n;
  }
};

void main() {}
